test/unit/lib/filterServices.js

```diff
@@ -65,8 +65,8 @@ describe('filterServices', () => {
     it('should return the nearest obj first', () => {
       const limits = { nearby: 3, open: 1 };
       const toFilter = [
-        { distanceInMiles: 2, identifier: 'A' },
-        { distanceInMiles: 1, identifier: 'B' },
+        { dist: 2, identifier: 'A' },
+        { dist: 1, identifier: 'B' },
```

Comment:
This doesn't need to be done if the query is changed to `distanceField: 'distanceInMiles'`
https://github.com/nhsuk/nearby-services-api/pull/30/files#diff-6b3c91a372d0bb755710f47f6b10c6e3R20